.gitignore

```diff
@@ -67,6 +67,10 @@ cmake-build-debug/
 generated/
 h3Config.cmake
 h3ConfigVersion.cmake
+CMakeDoxyfile.in
+CMakeDoxygenDefaults.cmake
+DartConfiguration.tcl
```

Comment:
This is a strange one to include in the gitignore.

Reply:
It's new in 3.11. From the release notes:

 * The "FindDoxygen" module "doxygen_add_docs()" function now
   supports a new "DOXYGEN_VERBATIM_VARS" list variable.  Any
   "DOXYGEN_..." variable contained in that list will bypass the
   automatic quoting logic, leaving its contents untouched when
   transferring them to the output "Doxyfile".